website/docs/r/privatelink_endpoint_service_data_federation_online_archive.html.markdown

```diff
@@ -33,13 +33,15 @@ resource "mongodbatlas_privatelink_endpoint_service_data_federation_online_archi
 * `endpoint_id` (Required) - Unique 22-character alphanumeric string that identifies the private endpoint. See [Atlas Data Lake supports Amazon Web Services private endpoints using the AWS PrivateLink feature](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Data-Federation/operation/createDataFederationPrivateEndpoint:~:text=Atlas%20Data%20Lake%20supports%20Amazon%20Web%20Services%20private%20endpoints%20using%20the%20AWS%20PrivateLink%20feature).
 * `provider_name` (Required) - Human-readable label that identifies the cloud service provider. 
 * `timeouts`- (Optional) The duration of time to wait for Private Endpoint Service to be created or deleted. The timeout value is definded by a signed sequence of decimal numbers with an time unit suffix such as: `1h45m`, `300s`, `10m`, .... The valid time units are:  `ns`, `us` (or `µs`), `ms`, `s`, `m`, `h`. The default timeout for Private Endpoint create & delete is `2h`. Learn more about timeouts [here](https://www.terraform.io/plugin/sdkv2/resources/retries-and-customizable-timeouts).
+* `region` - (Optional) Physical location of your MongoDB cluster. The region you choose can affect network latency for clients accessing your databases.  Requires the **Atlas region name**, see the reference list for [AWS](https://docs.atlas.mongodb.com/reference/amazon-aws/), [GCP](https://docs.atlas.mongodb.com/reference/google-gcp/), [Azure](https://docs.atlas.mongodb.com/reference/microsoft-azure/).
+* `customer_endpoint_dns_name` - (Optional) Human-readable label to identify customer's VPC endpoint DNS name. Must follow regex: `vpce-[a-z0-9]{17}-[a-z0-9]{8}.vpce-svc-[a-z0-9]{17}.(.*).vpce.amazonaws.com`
```

Comment:
is this regex imposed by us or from the backend? if so, could we reference the official documentation where this is validated as well?

Reply:
Yes, I learned this the "hard" way 😅 
I had to dig into the code to find it. The [API docs](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Data-Federation/operation/createDataFederationPrivateEndpoint)  didn't say anything.
I messaged Zuhair about it, but I didn't get a reply yet

Reply:
Made a few suggestions directly in this PR. Cleaned up example to make sure reader is aware of appriorapate formats for `customer_endpoint_dns_name` and `endpoint_id`. Also updated docs definitions to be more inline with Admin API.

Reply:
Changes made in 7c55d85 ✅